src/components/marker-table/ContextMenu.js

```diff
@@ -94,6 +121,24 @@ class MarkersContextMenu extends PureComponent<Props> {
     copy(markers[selectedMarker].name);
   };
 
+  copyMarkerCause = () => {
+    const {
+      thread,
+      implementationFilter,
+      markers,
+      selectedMarker,
+    } = this.props;
+    const marker = markers[selectedMarker];
+    if (marker.cause) {
```

Comment:
So first i'd like to apologize, because the cause is indeed where you got it before: in `data`.
So the code you could use is
```js
if (marker && marker.data && marker.data.cause) {
  const stack = this.convertStackToString(
      marker.data.cause.stack,
      thread,
      implementationFilter
    );
  ... copy stack
}
```

But this doesn't work because of how our typing is done. So I fixed our typing in https://github.com/devtools-html/perf.html/pull/1507, and with this PR, now this works as expected.

So you can wait than this PR lands and then rebase on top of master so that you can use it.

Reply:
https://github.com/devtools-html/perf.html/pull/1507 just landed, so I think you can try to move forward now. Thanks for your patience!

Reply:
okay :)